helpers/partial.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 const factory = globals => {
-    return function(name) {
-        const options = arguments[arguments.length - 1];
+    return function(name, options) {
+        options = options ? options : name;
```

Comment:
I think name should be required here.

Reply:
There are tests that say should not trigger an error if partial name is empty.